libc/test/src/__support/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ add_libc_test(
   DEPENDS
     libc.src.__support.high_precision_decimal
     libc.src.__support.uint128
```

Comment:
Put a FIXME here with the reason why it's disabled.

Reply:
sure